src/main/resources/environment.conf

```diff
@@ -20,6 +20,8 @@ processor {
   vehicleposition {
     maxTimeDifference = 5 seconds # Discard vehicle positions when timestamp differs from current time
     maxTimeDifference = ${?PROCESSOR_VEHICLE_POSITION_MAX_TIME_DIFFERENCE}
+    maxDelayAllowed = 10800 seconds # Discard vehicle positions when vehicle is delayed more than this. 0 seconds disables filtering.
```

Comment:
Maybe use zero by default to disable filtering?

Reply:
Default value was specified to be 3 hours in ticket, we can go with "disabled" also though and use as required, makes more sense probably